### ISSUE TITLE
Implement a collector for FCC broadband data to explore internet availability by location

### ISSUE THREAD

**Issue by MattTriano** [MattTriano/analytics_data_where_house#31]
A high-speed internet connection is a prerequisite for remote work/school, but [many blocks in the US](https://broadbandmap.fcc.gov/home) have no high-speed internet options as no telecom company has built the infrastructure needed to connect that block to a high speed network (yet?).

Telecom companies have to register infrastructure deployments with the FCC (through FCC Form 477) and some (or maybe all?) of that entered data can be downloaded through [this data access portal](https://broadbandmap.fcc.gov/data-download). It looks like files might follow consistently applied naming conventions (and if that's accurate, I'll be able to engineer URIs and automate collection).

It looks like the FCC [just changed](https://docs.fcc.gov/public/attachments/DA-22-1345A1.pdf) their broadband deployment data collection process. It's not clear if this will change the method for accessing data, but from [this Form 477 resources page](https://web.archive.org/web/20221225214345/https://www.fcc.gov/economics-analytics/industry-analysis-division/form-477-resources), the replacement system won't require telecoms to provide block-level deployment data. 

TLDR: I shouldn't prioritize building this until it's clear that the data will still have sufficient resolution to be useful.

Other Resources:
* [FCC Data Documentation](https://us-fcc.app.box.com/v/bdc-data-downloads-output)
* [FCC Databases](https://www.fcc.gov/licensing-databases/search-fcc-databases)
